.github/workflows/dev-release.yaml

```diff
@@ -0,0 +1,36 @@
+name: seqr dev release
+on:
+  workflow_run:
+    workflows: ["Unit Tests"]
+    types:
+      - completed
+    branches:
+      - seqr-helm-version # TODO: trigger this flow on dev
```

Comment:
should this TODO still be in here

Reply:
yes, because we're not going to trigger this workflow on changes to dev yet